TwoNote/app/src/main/res/layout/activity_main.xml

```diff
@@ -9,7 +9,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    app:show_in_dual_screen_start="@layout/fragment_items_list"
+    app:show_in_dual_screen_start="@layout/fragment_items_note"
     app:show_in_dual_screen_end="@layout/fragment_item_detail"
-    app:show_in_single_screen="@layout/fragment_items_list"
+    app:show_in_single_screen="@layout/fragment_items_note"
```

Comment:
should we keep the list as the default and change the `fragment_item_detail` one to the note instead? idk